add morph items

JaskRendix opened the pull request:
PR addresses the additions of morph items (booster + various):
- updated credits.md;
- created JSON files;
- updated PNGs inside the item folder;
- added the items + descriptions inside the PO file (some are missing);
- resorted items, there were items in other parts of the file, now everything is clustered in a section;
- updated evolve.py (related to evolution);
- added 2 item validator inside db.py (if the item exists or not); 1st in evolution, 2nd in economyitemmodel;
- reintroduced the evolution path == standard inside combat.py (nothing changes, but it allows only standard, in another PR the problem will be addressed, the "problem" concerns mixed and gender evolutions);
- updated Vivipere.json (still waiting about the fossil issue);

@Sanglorian 
before merging I have a question: [Vivipere](https://wiki.tuxemon.org/Vivipere) evolves into [Vividactil](https://wiki.tuxemon.org/Vividactil) through a fossil. Is it [Rhincus Fossil](https://wiki.tuxemon.org/Rhincus_Fossil), [Shammer Fossil](https://wiki.tuxemon.org/Shammer_Fossil), both or did you misspelled [Petrified Dung](https://wiki.tuxemon.org/Petrified_Dung)? I ask because until now no monster uses Petrified Dung for evolution. Let me know.

everything tested + black

Added items:
```
Booster Tech
Earth Booster
Fire Booster
Flintstone
Lucky Bamboo
Metal Booster
Miaow Milk
Peace Lily
Petrified Dung
Pyramidion
Sea Girdle
Stovepipe
Sweet Sand
Tectonic Drill
Thunderstone
Water Booster
Wood Booster
+
Greenwash badge
Nimrod badge
Omnichannel badge
Scoop badge
Shaft badge
```

JaskRendix committed:
morph items

JaskRendix committed:
Merge branch 'Tuxemon:development' into morph-items

Sanglorian commented:
Any of those three should work - the two fossils or the dung. Thanks for checking! 

The dung's main use is to turn into Coproblight, although as that's an unfinished mon it hasn't been implemented yet.

JaskRendix commented:
No problem. updated the vivipere.json and added the two fossils too. Thanks again.

Updated also some existing items as KeyItem, source: https://wiki.tuxemon.org/Category:Item

JaskRendix committed:
morph items
